gelf/tcpwriter.go

```diff
@@ -87,3 +63,19 @@ func (w *TCPWriter) Write(p []byte) (n int, err error) {
 
 	return len(p), nil
 }
+
+func (w *TCPWriter) writeToSocketWithReconnectAttempts(zBytes []byte) (n int, err error) {
+	var errConn error
+	w.mu.Lock()
+	for i := 0; n <= 3; i++ {
```

Comment:
Yeah, that's a problem. Let's skip that one for now. Maybe we see a solution in a later stage.